src/j1client.js

```diff
@@ -10,6 +10,19 @@ const fetch = require("node-fetch").default;
 
 const J1_USER_POOL_ID_PROD = "us-east-2_9fnMVHuxD";
 const J1_CLIENT_ID_PROD = "1hcv141pqth5f49df7o28ngq1u";
+const REQUEST_TIMEOUT_IN_MS = 1000 * 60 * 5; // 5 minute timeout.
```

Comment:
How about call this `QUERY_RESULTS_TIMEOUT` and add a comment that indicates that this is the max amount of time that we will wait for query results. "request timeout" seems to imply that it is timeout for individual request.

Reply:
Ok.
Is it helpful to give the user some sort of prompt in the command line ("Query started at 2020-03-19T20:07:06.365Z")?

Reply:
This library is being used as a CLI tool and as a package used by other applications so it wouldn't make any sense to add logging like this.

Reply:
@philidem updated